docs/src/test/java/jdocs/stream/operators/SourceOrFlow.java

```diff
@@ -608,6 +608,20 @@ void filterNotExample() {
     // #filterNot
   }
 
+  void distinctUntilChangedExample() {
+    // #distinctUntilChanged
+    Source.from(Arrays.asList(1, 2, 2, 3, 3, 4))
+        .distinctUntilChanged()
+        .runForeach(System.out::println, system);
+    // prints:
+    // 1
+    // 2
+    // 3
+    // 4
```

Comment:
Can we add another '2' to the example to show what happens when a value shows up that has been seen before, but was not the most recent value?

Reply:
Do you mean `1,2,2,3,3,2,4`?

Reply:
Yes, for example!